api/routes.py

```diff
@@ -11,7 +11,7 @@
 from api.payload import Payload
 
 try:
-    from api.secrets import TOKENS
+    import api.secrets
```

Comment:
We have to change this import because mock patches references to the object, not the object itself. So from mock's perspective, these two imports are different, and the first (`from api.secrets import TOKENS`) is a different object.